### PR TITLE
Improved Logging for NIC Drop Scenario

### PR DESCRIPTION
**Pull request by nibanks**
## Description

Improves the logging for the NIC drop scenario we've been investigating.

## Testing

Perf pipeline

## Documentation

N/A

### CODE REVIEW

File: .azure/azure-pipelines.perf.yml [hunk 1]
```diff
@@ -353,14 +354,15 @@ stages:
...
-  condition: succeededOrFailed()
-  dependsOn:
-  - perf_winkernel
-  - perf_winuser_schannel
-  - perf_winuser_xdp
-  - perf_winuser_openssl
-  - perf_linux_openssl
-  jobs:
-  - template: ./templates/post-process-performance.yml
+- ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI') }}:
```
Comment (nibanks) [inline .azure/azure-pipelines.perf.yml:357]: Necessary for manual perf runs.

File: src/manifest/MsQuic.wprp [hunk 2]
```diff
@@ -20,6 +20,22 @@
...
         <Keyword Value="SampledProfile"/>
+        <Keyword Value="DPC"/>
+        <Keyword Value="Interrupt"/>
+      </Keywords>
+      <Stacks>
+        <Stack Value="SampledProfile"/>
+      </Stacks>
+    </SystemProvider>
+
+    <SystemProvider Id="SP_SystemThreadExecutionMedium">
```
Comment (nibanks) [inline src/manifest/MsQuic.wprp:31]: Remove

File: docs/Diagnostics.md [hunk 3]
```diff
@@ -99,6 +99,7 @@ wpr.exe -start MsQuic.wprp!PROFILE -filemode
 You must replace `PROFILE` with the name of the profile you want to run. Some of the useful profiles are:
 
 - `Stacks.Light`- Collects CPU callstacks.
+- `Stacks.Verbose`- Collects CPU callstacks, DPUs and interrupts.
```
Comment (mtfriesen) [inline docs/Diagnostics.md:102]: DPCs instead of DPUs, right?

**Review by mtfriesen — APPROVED** [review]

File: docs/Diagnostics.md (hunk 3 above)
Comment (nibanks) [reply docs/Diagnostics.md:102]: Yes.

**Pull request merged**